app/views/pages/faq.html.erb

```diff
@@ -49,6 +49,7 @@
       <b>How do I change my Twitter/GitHub username?</b>
       <br>
       Email yo@dev.to and we'll take care of it.
+      If you are authenticating with the other one of the two, you can also head to <a href="https://dev.to/settings/account">Settings &gt; Account</a> and remove and then reässociate the one that you want to change yourself.
```

Comment:
I'm not sure I understand the verbiage in this change, perhaps it can be simplified to something like:

`You can also remove your associated account and re-authenticate in your <a href="https://dev.to/settings/account">Account Settings</a>.`

Reply:
There is an A-diaeresis that shouldn't be here: `reässociate`.

Reply:
> There is an A-diaeresis that shouldn't be here: reässociate.

It indicates that both vowels are pronounced independently, not as a dipthong. I think the only English language publication where I regularly see this is the New Yorker (e.g. coöperate).

Reply:
TIL

Reply:
> > There is an A-diaeresis that shouldn't be here: reässociate.
> 
> It indicates that both vowels are pronounced independently, not as a dipthong. I think the only English language publication where I regularly see this is the New Yorker (e.g. coöperate).

Recently I have been typing a lot of stuff with <q>æ</q> not long ago and somehow confused it for a diphtong in the English language.

Reply:
> I'm not sure I understand the verbiage in this change, perhaps it can be simplified to something like:
> 
> `You can also remove your associated account and re-authenticate in your <a href="https://dev.to/settings/account">Account Settings</a>.`

I believe we should highlight that another association must be present, and that this other association will become the only way of authentication — so it must be taken care of not to lose that other associated account… especially to make sure that even the right one is connected. Also to differentiate that from the "email us" preceding option i believe it is important to highlight the <q>yourselfness</q> of this action. Also I believe using <q>reassociate</q> is a better choice since <q>reauthenticate</q> might be misunderstood; maybe <q>reauthenticate **it**</q>, but I'm still not sure about it.